Dockerfile

```diff
@@ -3,24 +3,38 @@ FROM nerc/spark-core
 LABEL maintainer "gareth.lloyd@stfc.ac.uk"
 
 ENV ZEPPELIN_VER 0.7.2
-ENV ZEPPELIN_HOME=/opt/zeppelin
+ENV ZEPPELIN_HOME /opt/zeppelin
+ENV ZEPPELIN_USER datalab
+ENV ZEPPELIN_UID 1000
 
-RUN mkdir -p /opt
+USER root
 
+# Install Zeppelin
 RUN wget -O /tmp/zeppelin-${ZEPPELIN_VER}-bin-all.tgz http://archive.apache.org/dist/zeppelin/zeppelin-${ZEPPELIN_VER}/zeppelin-${ZEPPELIN_VER}-bin-all.tgz && \
 	tar -zxvf /tmp/zeppelin-${ZEPPELIN_VER}-bin-all.tgz && \
 	rm -rf /tmp/zeppelin-${ZEPPELIN_VER}-bin-all.tgz && \
 	mv /zeppelin-${ZEPPELIN_VER}-bin-all ${ZEPPELIN_HOME}
 
-# Install GDAL utilities
-RUN apt-get install -y gdal-bin
+# Add datalab user
+RUN useradd -m -s /bin/bash -N -u $ZEPPELIN_UID $ZEPPELIN_USER && \
+	chown -R $ZEPPELIN_USER $ZEPPELIN_HOME
 
-EXPOSE 8080 8443
+# Install Tini
```

Comment:
Is tini going to be in the Spark image too? If so, we might want to move this bit to the base image.